content/python/concepts/built-in-functions/terms/map/map.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'map()'
+Description: 'Returns an iterator that takes a function and applies it to every item in an iterable.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+'A built-in function that takes a function and applies it to every item in an iterable. The output of `map()` is a map object, in the examples below `list()` is applied to the map expression to return a new list.'
+
+## Syntax
+
+```psuedo
+map(func_name, iterable)
+```
+
+The `func_name` is the function to be applied to the iterable and can be a lambda function or the name of any defined function. The `iterable`, such as a list, contains the items the function will act on.
+
+## Example 1
+
+Use `map()` to return the length of each string in the following list:
+
+```py
+some_strings = ['learn', 'how', 'to', 'code', 'online']
+
+print(list(map(len, some_strings))
+
+# Output: [5, 3, 2, 4, 6]
+```
+
+## Example 2
+
+The `map()` function can be used to transform all the list items to integers:
+
+```codebyte/python
+a_list = [1.6, 5.4,12.0]
+
+print(list(map(lambda x: int(x), a_list)))
+
```

Comment:
```suggestion
print(list(map(lambda x: int(x), a_list)))
```

Reply:
Sorry, I don't see the suggested change here. Please confirm

Reply:
Delete line # 47, please.